### PR TITLE
Changelog for master

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @craigcomstock can review this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,71 @@
...
 Notable changes to the framework should be documented here
 
+3.13.0:
+	- Add 'system-uuid' to default dmidecode inventory (CFE-2925)
+	- Add inventory of AWS EC2 linux instances (CFE-2924)
+	- Add ubuntu 18 to package map for self upgrade (ENT-4118)
+	- Allow dmidefs inventory to be overridden via augments (CFE-2927)
+	- Also list  packages updates for hold packages: (CFE-2855)
+	- Analyze yum return code before parsing its output (CFE-2868)
+	- Avoid promise not kept (ENT-3965)
```
Comment (vpodzime) [inline CHANGELOG.md]: More details needed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,71 @@
...
+	- Clear __lastseenhostslogs every 5 minutes. (ENT-3550)
+	- Configure Enterprise hub pull collection schedule via augments
+	  (ENT-3834)
+	- Configure agent_expireafter from augments (ENT-4308)
+	- Create desired version tracking data when necessary (ENT-3937)
+	- Cron based watchdog for cf-execd on AIX (ENT-3963)
+	- Detect systemd service enablement for non native services (CFE-2932)
+	- Document how def.acl is used and how to configure it (CFE-2861)
+	- Fix augments control state paths to work on windows (ENT-3839)
+	- Fix compatibility with 3.7.x (ENT-3964)
```
Comment (vpodzime) [inline CHANGELOG.md]: Should be dropped.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,71 @@
...
+	- Fix name of tunable to control max client side report history
+	  (CFE-2926)
+	- Fix package_latest detecting larger version in some cases (CFE-1743)
+	- Fix policy backward compatibility (ENT-3910)
+	- Fix policy compatibility for 3.7.x (ENT-3907)
```
Comment (vpodzime) [inline CHANGELOG.md]: The two lines above also should be dropped.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,71 @@
...
+	  (ENT-4092)
+	- Remove Design Center related policies
+	  Design center never left beta and has been deprecated. Supporting policies have
+	  been removed. If you wish to continue using design center sketches you must
+	  incorporate them into inputs and the bundlesequence manually.
+	  (ENT-4050)
+	- Remove more smartquotes (ENT-3823)
+	- Remove templates for deprecated components (ENT-3781)
+	- Remove un-necessary agent run during self upgrade (ENT-4116)
+	- Replace unicode smartquotes with apostrophe (ENT-3823)
```
Comment (vpodzime) [inline CHANGELOG.md]: The two entries about smartquotes should be merged into a single one.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,68 @@
...
+	- Also list  packages updates for hold packages: (CFE-2855)
+	- Analyze yum return code before parsing its output (CFE-2868)
+	- Fixed issue when promise to edit file that does not exist caused "promise
+	  not kept" condition (ENT-3965)
+	- Avoid trying to read /proc/meminfo when it doesn't exist (CFE-2922)
+	- Avoid use of $(version) for package_version in legacy implementation
+	  (ENT-3963)
+	- Cleanup old report data relative to the most recent changetimestamp
+	  (ENT-4807)
+	- Clear __lastseenhostslogs every 5 minutes. (ENT-3550)
```
Comment (olehermanse) [inline CHANGELOG.md]: Put backticks around `__lastseenhostslogs` to avoid markdown boldness.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,68 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.13.0:
+	- Add 'system-uuid' to default dmidecode inventory (CFE-2925)
+	- Add inventory of AWS EC2 linux instances (CFE-2924)
+	- Add ubuntu 18 to package map for self upgrade (ENT-4118)
+	- Allow dmidefs inventory to be overridden via augments (CFE-2927)
+	- Also list  packages updates for hold packages: (CFE-2855)
```
Comment (olehermanse) [inline CHANGELOG.md]: Doesn't make sense without more detail.

**Comment by Lex-2008** [comment]
don'!

**Comment by Lex-2008** [comment]
fix'd!

**Comment by nickanderson** [comment]
Maybe squash the additional commits from me changing it directly :)

**Pull request merged**